src/docs/sub-activate.md

```diff
@@ -40,21 +40,25 @@ For this step, you must have a sufficient amount of XRT tokens (minimum of 2-3 X
 
 Now you need to setup you subscription by adding the `CONTROLLER` account to it.
 
-{% roboWikiPicture {src:"docs/home-assistant/sub-setup.png", alt:"sub_setup"} %}{% endroboWikiPicture %}
+{% roboWikiPicture {src:"docs/home-assistant/sub-download-backup.png", alt:"sub_setup"} %}{% endroboWikiPicture %}
 
 1. Go to the Robonomics dApp and navigate to the [setup a subscription page](https://robonomics.app/#/rws-setup). Navigate to the **Subscription settings** section.
 
-2. In the `Controller's seed phrase` field press magic wand to create new `CONTROLLER` account.
+2. Click on `DOWNLOAD BACKUP` and select `FOR THE SERVER` option. 
+
+{% roboWikiNote {type: "warning", title: "Important information" }%} This action will create a new controller for your subscription. Do not forget to add it to the subscription. {% endroboWikiNote %}
 
 3. In the pop-up create password for the `CONTROLLER` account.
 
-4. On the next pop-up, you will see the address of your new account and the mnemonic seed phrase. Save the mnemonic seed phrase securely because you will need it later 
-for integration setup. Additionally, the JSON file with the `CONTROLLER` account will be downloaded. You can import it to your wallet. 
-How to do it for Polkadot.js extension can be found [here](/docs/create-account-in-dapp/).
+{% roboWikiPicture {src:"docs/home-assistant/server-new-settings.png", alt:"controller create"} %}{% endroboWikiPicture %}
+
+4. In the next pop-up, you will see the address of your new account and the mnemonic seed phrase. Save the mnemonic seed phrase securely. In the downloads folder, you will find two JSON files: the first file is named `Controller-<address>.json`, where `<address>` is the address of your newly generated controller. The second file is named `robonomics.app-settings-<subscirption-name>-server.json`, where `<subscirption-name>` is the name of your subscription. Save these files securely, as they will be needed later for the integration setup. Additionally, you can import your controller account into your wallet. Instructions for importing it into the Polkadot.js extension can be found [here](/docs/create-account-in-dapp/).
+
+5. (Optional) You can add credentials for pinning service Pinata or other custom gateway to spread your data wider over the IPFS network.
 
-{% roboWikiPicture {src:"docs/home-assistant/controller-create.jpg", alt:"controller create"} %}{% endroboWikiPicture %}
+{% roboWikiNote {title:"Note", type: "Note"}%} In [Pinata Setup section](/docs/pinata-setup) you can find more detailed information about using Pinata.{% endroboWikiNote %}
 
-5. Close pop-up and click the `SAVE` button.
+6. Close pop-up and click the `SAVE` button.
 
 ## Add Controller Account to Subscription
```

Comment:
Need to provide the variant how to copy the **Controller** address from setup page, if the user don't add  **Controller** account to extension.